pkg/aws/services/iam/oidc.go

```diff
@@ -350,7 +350,11 @@ func caThumbPrints(ep string) ([]string, error) {
 		if err != nil {
 			return nil, microerror.Mask(errors.Wrapf(err, "failed to get %s", ep))
 		}
-		defer resp.Body.Close()
+		defer func() {
+			if err := resp.Body.Close(); err != nil {
+				s.scope.Logger().Error(err, "failed to close response body", "ep", ep)
+			}
+		}()
```

Comment:
This feels really silly. I'm not sure if it's worth doing this.

Reply:
Agree, sounds really silly, but it's ok I guess.